app/org/sagebionetworks/bridge/dynamodb/DynamoTaskEvent.java

```diff
@@ -77,18 +86,19 @@ public Builder withEventType(TaskEventType type) {
             this.eventType = type;
             return this;
         }
-        public Builder withValue(String value) {
-            this.value = value;
+        public Builder withAnswerValue(String answerValue) {
+            this.answerValue = answerValue;
             return this;
         }
         private String getEventId() {
             if (type == null) {
                 return null;
             }
+            if (type == TaskEventObjectType.ENROLLMENT) {
+                return type.name().toLowerCase();
+            }
             String typeName = type.name().toLowerCase();
-            if (objectId != null && eventType != null && value != null) {
```

Comment:
Enrollment is an event that's not tied to an entity (an object like survey or survey answer). So, yes.